gulpfile.js/tasks/rev/rev-css.js

```diff
@@ -11,7 +11,7 @@ var uglify = require('gulp-uglify')
 gulp.task('rev-css', function(){
   return gulp.src(path.join(config.root.dest,'/**/*.css'))
     .pipe(rev())
-    .pipe(cssnano())
+    .pipe(cssnano({autoprefixer: false}))
```

Comment:
Oh, good catch. Thanks! But Maybe I should just remove autoprefixer from the gulp task...